login-workflow/src/components/Spinner.tsx

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
-import { CircularProgress, makeStyles, createStyles } from '@material-ui/core';
+import { CircularProgress, makeStyles, createStyles, useTheme, Theme } from '@material-ui/core';
 
-const useStyles = makeStyles(() =>
+const useStyles = makeStyles((theme: Theme) =>
     createStyles({
         background: {
-            position: 'absolute',
+            position: 'fixed',
             top: 0,
             left: 0,
             right: 0,
             bottom: 0,
             zIndex: 1000,
-            backgroundColor: 'rgba(255,255,255,0.6)',
+            backgroundColor: theme.palette.type === 'light' ? 'rgba(255,255,255,0.6)' : 'rgba(0,0,0,0.7)',
```

Comment:
Were we going to change this black (0,0,0) to the dark black color from the dark theme values for the scrim?

Reply:
Not sure what you mean here. Dark black 900 is #0000000 / rgb(0,0,0).

Reply:
![image](https://user-images.githubusercontent.com/29152776/121402238-a1c6cd00-c927-11eb-9e47-40140c06979f.png)

I did not realize DarkBlack[900] was pure black...my bad!